src/Tribe/Cache.php

```diff
@@ -87,7 +87,11 @@ public function set( $id, $value, $expiration = 0, $expiration_trigger = '' ) {
 	 * @return bool
 	 */
 	public function set_transient( $id, $value, $expiration = 0, $expiration_trigger = '' ) {
-		return set_transient( $this->get_id( $id, $expiration_trigger ), $value, $expiration );
+		if ( $this->data_size_over_packet_size( $value, $serialized_value ) ) {
```

Comment:
Not seeing $serialized_value set up here

Reply:
No need to. It's set by reference, like the `$matches` in `preg_match`.

Reply:
You can test this out on a PHP 5.6 machine with this:

```bash
php -r 'preg_match("/foo/", "foo bar", $m) && print_r($m);'
```

Reply:
OK so there's no PHP notices calling an undefined variable at this line then?

Reply:
I'm double-checking to make sure, spinning up now.

Reply:
Confirmed, no errors on a PHP 5.6 machine and 7.2 one.

Reply:
✅